.docker/ci/Dockerfile

```diff
@@ -1,50 +1,30 @@
-# moveit/moveit:melodic-ci
-# Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
 
-FROM ros:melodic-ros-base
-MAINTAINER Dave Coleman dave@picknik.ai
+MAINTAINER Lander Usategui  <lander@erlerobotics.com>
```

Comment:
I'm not sure if this should change, @davetcoleman ?

Reply:
No problem with this, I'll keep Dave as maintainer.

Reply:
FYI that field is deprecated:  
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Reply:
Thanks @ruffsl!

Reply:
i do not need to be a maintainer of this anymore